packages/diagrams/backend/sirius-components-diagrams-tests/src/main/java/org/eclipse/sirius/components/diagrams/tests/navigation/DiagramNavigator.java

```diff
@@ -105,6 +106,10 @@ public EdgeNavigator edgeWithEdgeDescriptionId(String edgeDescriptionId) {
         return new EdgeNavigator(edges.get(0), this.cache);
     }
 
+    public Collection<Node> findAllNodes() {
+        return this.cache.getIdToNode().values();
+    }
```

Comment:
I used this PR as an opportunity to add this method that would be helpful for some downstream tests.